packages/hub/node-tests/tasks/drop-card-test.ts

```diff
@@ -0,0 +1,128 @@
+
+  class StubRelayService {
+    async provisionPrepaidCardV2(userAddress: string, requestedSku: string) {
+      provisionPrepaidCardCalls++;
+
+      if (provisioningShouldError) {
+        throw new Error('provisioning should error');
+      }
+
+      provisionedAddress = userAddress;
+      provisionedSku = requestedSku;
+      return Promise.resolve(mockTxnHash);
+    }
+  }
+
+  this.beforeEach(async function () {
+    registry(this).register('relay', StubRelayService, { type: 'service' });
+    provisionPrepaidCardCalls = 0;
+    provisioningShouldError = false;
+
+    Sentry.init({
+      dsn: DUMMY_DSN,
+      release: 'test',
+      tracesSampleRate: 1,
+      transport: sentryTransport,
+    });
+    testkit.reset();
+  });
+
+  let { getContainer } = setupHub(this);
+
+  this.beforeEach(async function () {
+    emailCardDropRequestQueries = await getContainer().lookup('email-card-drop-requests', { type: 'query' });
+    await emailCardDropRequestQueries.insert(unclaimedEoa);
+  });
+
+  it('calls the relay service to provision a prepaid card', async function () {
+    let task = (await getContainer().lookup('drop-card')) as DropCardTask;
+
+    await task.perform({
+      id: unclaimedEoa.id,
+    });
+
+    expect(provisionPrepaidCardCalls).to.equal(1);
+    expect(provisionedAddress).to.equal(unclaimedEoa.ownerAddress);
+    expect(provisionedSku).to.equal(sku);
+
+    let requests = await emailCardDropRequestQueries.query({ id: unclaimedEoa.id });
+    let request = requests[0];
+
+    expect(request.transactionHash).to.equal(mockTxnHash);
+  });
+
+  it('logs an error and does not call the relay service when the request is not found', async function () {
+    let task = (await getContainer().lookup('drop-card')) as DropCardTask;
+    let nonexistentUuid = '9abdd42e-abcc-4ce2-b4f8-e6a04efdc5ec';
+
+    await task.perform({
+      id: nonexistentUuid,
+    });
+
+    expect(provisionPrepaidCardCalls).to.equal(0);
+
+    await waitFor(() => testkit.reports().length > 0);
+
+    expect(testkit.reports()[0].error?.message).to.equal(
+      `Could not find email card drop request with id ${nonexistentUuid}`
+    );
+    expect(testkit.reports()[0].tags).to.deep.equal({
+      action: 'drop-card',
+    });
+  });
+
+  it('logs an error when the relay server call fails', async function () {
+    provisioningShouldError = true;
+
+    let task = (await getContainer().lookup('drop-card')) as DropCardTask;
+
+    await task.perform({
+      id: unclaimedEoa.id,
+    });
+
+    expect(provisionPrepaidCardCalls).to.equal(1);
+
+    await waitFor(() => testkit.reports().length > 0);
+
+    expect(testkit.reports()[0].error?.message).to.equal('provisioning should error');
+    expect(testkit.reports()[0].tags).to.deep.equal({
+      action: 'drop-card',
+    });
+
+    let requests = await emailCardDropRequestQueries.query({ id: unclaimedEoa.id });
+    let request = requests[0];
+
+    expect(request.transactionHash).to.be.null;
```

Comment:
After adding the change for `claimedAt`, I think we should have a test case that asserts that `claimedAt` is set to `null` when it was set to a `Date` at the beginning of the test. Maybe all the tests cases here should have `claimedAt` set already?